<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SatelliteSpotWindow</class>
 <widget class="QWidget" name="SatelliteSpotWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>967</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo satellite spot. task 5290, 5530_5406.</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="SatelliteMapWidget" name="satelliteMapWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnOpenMap">
           <property name="text">
            <string>open map</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStartTimer">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>start timer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStopTimer">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>stop timer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon0Lat0">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=0,lat=0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon0Lat35">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=0,lat=35</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon90Lat0">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=90,lat=0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon0Lat40">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=0,lat=40</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon0LatM40">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=0,lat=-40</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon170Lat0">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=170,lat=0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon180Lat0">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=180,lat=0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLonM150Lat0">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=-150,lat=0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon180Lat6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=180,lat=6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLonM180MLat6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=-180,lat=-6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon180Lat60">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=180,lat=60</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon170Lat60">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=170,lat=60</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLonM180LatM60">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=-180,lat=-60</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLonM170LatM60">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=-170,lat=-60</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLon170Lat45">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>lon=170,lat=45</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="checkBoxMoveSatellite">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>move satellite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSpotPoints">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>draw spot points</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSpotPolygon">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>draw spot polygon</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClearAll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>92</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>clear all</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lblSatLon">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>117</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>117</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>satellite longitude, deg:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxSatLon">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>178</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horSliderSatLon">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblSatLat">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>117</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>117</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>satellite latitude, deg:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxSatLat">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>-90</number>
         </property>
         <property name="maximum">
          <number>90</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horSliderSatLat">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>-90</number>
         </property>
         <property name="maximum">
          <number>90</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lblSatAlt">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>117</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>117</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>satellite altitude, km:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxSatAlt">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>327</number>
         </property>
         <property name="maximum">
          <number>35786</number>
         </property>
         <property name="value">
          <number>35786</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horSliderSatAlt">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>327</number>
         </property>
         <property name="maximum">
          <number>35786</number>
         </property>
         <property name="value">
          <number>35786</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="txtLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SatelliteMapWidget</class>
   <extends>QWidget</extends>
   <header>SatelliteMapWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
